apps/base/migrations/0003_auto_20160216_2212.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
```

Comment:
So apparently it's a best practice to push migrations to have a history of database changes in git and to make sure the db is synced with the model for each person developing. 

My source is pretty much stackoverflow, so I could be wrong. Sources are [here](http://stackoverflow.com/questions/10234956/is-it-a-good-idea-to-keep-database-migration-inside-vcs) and [here](http://stackoverflow.com/questions/32376092/why-there-is-need-to-push-django-migrations-to-version-control-system).